docs/guides/1227-contributing.md

```diff
@@ -172,3 +172,13 @@ git commit --amend -s
 # Force push the new commit to re-run all GitHub Actions jobs:
 git push -f mybranch
 ```
+
+### Can I use a remote development environment?
+
+Yes! The Dagger repository has Github Codespaces configuration included to help help you get started contributing directly from GitHub.
```

Comment:
- Embed a link to Dagger repository 
- Github -> GitHub
- You repeated `help` 2 times
- To help you contributing directly from GitHub